test.sh

```diff
@@ -32,11 +32,23 @@ killall secretshare-server
 ./build/$CURRENT_OS-$CURRENT_ARCH/secretshare-server -config secretshare-server.json &> test-server.log &
 server_pid=$!
 
+if [ "x$server_pid" == "x" ]; then
+    echo 'Failed to start server!'
+    exit 1
+fi
+
 sleep 2
 
+if ! ps aux | grep $server_pid; then
+    echo 'Server died unexpectedly!'
+    exit 1
+fi
+
 CLIENT="./build/$CURRENT_OS-$CURRENT_ARCH/secretshare --endpoint http://localhost:8080 --bucket-region $TEST_BUCKET_REGION --bucket $TEST_BUCKET"
 
-export SECRETSHARE_KEY=$(get_key_from_vars)
+export SECRETSHARE_KEY=$(get_key_from_server_config)
+echo "SECRETSHARE_KEY=$SECRETSHARE_KEY"
```

Comment:
Whoops, no.  I think these were the only ones.  I'll fix this and update the PR in a moment.